### ISSUE TITLE
Setting theme to auto causes container to restart endlessly

### ISSUE THREAD

**Issue by raspberrycoulis** [authelia/authelia#2237]
### Description

According to the [documentation regarding themes](https://www.authelia.com/docs/configuration/theme.html), you should be able to set the theme to `auto` by adding the following to the `configuration.yml`:
```yaml
theme: auto
```
However, doing so causes the Docker container to restart endlessly. Setting the theme to either `light`, `grey` or `dark` works as expected, but only the `auto` setting fails.

### Expected Behaviour

The theme should be set according to the user's device preferences - i.e. light during the day and dark during the night.

### Reproduction Steps

1. Edit `configuration.yml` to include `theme: auto`
2. Restart container - i.e. `docker container restart authelia`
3. In my case, I also need to restart Traefik with `docker container restart traefik`
4. If the theme is set to `auto`, the Authelia container just restarts endlessly, but setting to `light`, `grey` or `dark` works just fine.

### Additional Information

I'm running this on a Raspberry Pi 4 (8GB model) and my browser is Brave 1.26.77 Chromium: 91.0.4472.164 (Official Build) (x86_64) on macOS 11.5.1.

**Comment by james-d-elliott** [comment]
This is just due to our documentation being pulled from master which is ahead of the latest release. When 4.30.0 comes out it will be supported. Apologies for the confusion, we're actively looking into versioned docs.

The log messages from the container would reveal auto is not actually supported on 4.27.4.

**Comment by raspberrycoulis** [comment]
> The log messages from the container would reveal auto is not actually supported on 4.27.4.

I never saw the logs as I saw the new `theme: auto` option when reading the docs (I'm very new to Authelia), so just added that to my config then restarted my Authelia container. I just went with the process of elimination by trying all the options available (for the theme) and only the `auto` option didn't work.

I'll keep an eye open for the release of v4.30.0 then, but wanted to mention this as it caused confusion for me as a new user.
